pages/api/companies.ts

```diff
@@ -111,6 +110,13 @@ async function handlePost(req: NextApiRequest, res: NextApiResponse<ApiResponse<
     .json(createJsonResponse(newCompany, messages[MessageType.COMPANY_CREATED_SUCCESSFULLY]));
 }
 
+function parseGetQueryParams(req: NextApiRequest): CompanyQueryParams {
+  const { companyNames } = req.query;
+  const names =
+    companyNames === undefined ? [] : Array.isArray(companyNames) ? companyNames : splitByWhitespaces(companyNames);
+  return { companyNames: names };
```

Comment:
I think its better to require the companyName to be of type `arrays | undefined`, rather than allowing for split by whitespace. We'd have to then consider issues like "National University of Singapore" treated as 4 different company names.

Reply:
![image](https://user-images.githubusercontent.com/53614454/190133300-8e8635d1-89be-4b5c-8790-a30e7a564ffe.png)

Its considered as one. I don't get what you mean here.